### ISSUE TITLE
Date Input Field Label Should Be Read By Screen Reader

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1687]
When using a screen reader, the label for the date input field (e.g. start and end date) should be read when the field is focussed when you select it so you know the context.
